### PR TITLE
Improve client-side language handling

### PR DESCRIPTION
**Pull request by itshouldntdothis**
Store language locale in the html lang attribute to help client-side translation and only load required locale files.

### CODE REVIEW

File: static/js/eve-ui-config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var eveui_imageserver = function(image_ref) {
 	return eve_image(encodeURI(image_ref), 'png');
 };
 // fix for getting names for modules from ccp api that are in the users browser language
-eveui_accept_language = document.getElementById('lang-code').getAttribute('content');
+eveui_accept_language = document.getElementById('htmltag').getAttribute('lang');
```
Comment (SpeedProg) [inline static/js/eve-ui-config.js]: may be an idea to make this line:
```suggestion
eveui_accept_language = document.documentElement.getAttribute('lang');
```

File: templates/base.html [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
-<html lang="">
+<html id="htmltag" lang="{{ lang_code }}">
```
Comment (SpeedProg) [inline templates/base.html]: Also if the other code is accepted change this:
```suggestion
<html lang="{{ lang_code }}">
```

Comment (itshouldntdothis) [reply templates/base.html]: https://github.com/SpeedProg/eve-inc-waitlist/pull/12/commits/9f9443dea7c5098f7c8bea6aafe3037718fd7735 resolves this.

File: static/js/eve-ui-config.js (hunk 1 above)
Comment (itshouldntdothis) [reply static/js/eve-ui-config.js]: https://github.com/SpeedProg/eve-inc-waitlist/pull/12/commits/9f9443dea7c5098f7c8bea6aafe3037718fd7735 resolves this.

**Pull request merged**